packages/@expo/cli/src/start/doctor/dependencies/ensureDependenciesAsync.ts

```diff
@@ -112,17 +110,7 @@ export function createInstallCommand({
     version?: string | undefined;
   }[];
 }) {
-  return (
-    'npx expo install ' +
-    packages
-      .map(({ pkg, version }) => {
-        if (version) {
-          return [pkg, version].join('@');
-        }
-        return pkg;
-      })
-      .join(' ')
-  );
+  return 'npx expo install ' + packages.map(({ pkg }) => pkg).join(' ');
```

Comment:
It seems more useful to see the version info if we have it, otherwise the way you see is by installing and checking the diff.

Reply:
why do they need to see the version @EvanBacon? in other places we don't even print this, we prompt to install (eg: for typescript)

Reply:
The versions are still in the error message on the line before. Just not the install command.